drivers/pinctrl/intel/pinctrl-intel.c

```diff
@@ -1318,6 +1358,11 @@ int intel_pinctrl_probe(struct platform_device *pdev,
 	pctrl->soc = soc_data;
 	raw_spin_lock_init(&pctrl->lock);
 
+	if (dmi_first_match(dmi_retain_hostown_table)) {
+		pctrl->quirks |= PINCTRL_QUIRK_KEEP_HOSTOWN;
+		dev_dbg(&pdev->dev, "enabling KEEP_HOSTOWN quirk on this hw\n");
```

Comment:
```suggestion
		dev_info(&pdev->dev, "enabling KEEP_HOSTOWN quirk on this hw\n");
```
Please change this to info level, `dev_dbg` is only printed when dynamic debug is enabled. We want this to be printed every time this quirk is active to help detect any possible fallouts.